### ISSUE TITLE
Update requirements.txt

### ISSUE THREAD

**Issue by auria** [openstack-charmers/zaza#456]
`tox` environments fail for two reasons. The first one is because of `distro-info` requirement:

```
ERROR: Could not find a version that satisfies the requirement distro-info (from versions: 0.10)
ERROR: No matching distribution found for distro-info
```

Once the requirement is commented out (in my case, I have a deb pkg installed, `python3-distro-info`), `pip` runs for a very long time (I didn't wait until it finished) and the logs showed messages like the following:

```
INFO: This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. If you want to abort this run, you can press Ctrl + C to do so. To improve how pip performs, tell us what happened here: https://pip.pypa.io/surveys/backtracking
```

Per the latest version of the `classic-zaza` requirements.txt file, `pbr==5.6.0`. By applying this change, I can run  `pep8,py3` tox environments at a more normal speed.

**Comment by auria** [comment]
This issue is going to be fixed by @ajkavanagh at https://github.com/openstack-charmers/zaza/pull/454

The PR can be rejected and the issue linked with Alex' change.

**Comment by ajkavanagh** [comment]
> This issue is going to be fixed by @ajkavanagh at #454
> 
> The PR can be rejected and the issue linked with Alex' change.

#454 is going to be a while before it lands!  (it's a bit of a work-in-progress); I will take a look at #457 as it will un-break the tox.  Thanks for the patch.

**Comment by auria** [comment]
The initial PR was approved so this problem is fixed.
